src/components/atoms/BadgePreview/BadgePreviewProps.ts

```diff
@@ -0,0 +1,21 @@
+import { BadgeCategories, BadgeTypes } from '@businessLogic/badge'
+
+export type BadgePreviewEffects = 'wobble' | 'grow' | 'glare'
+export type BadgeSize = 'small' | 'medium' | 'large' | 'x-large'
+export type BadgeTextContrast = 'light' | 'dark' | 'light-withTextBackground' | 'dark-withTextBackground'
+export interface BadgePreviewProps {
+  size: BadgeSize
+  badgeCategory: BadgeCategories
+  badgeType: BadgeTypes
```

Comment:
Why do we need these props? (`badgeType` and `badgeCategory`) Right now, we don't have different designs based on it and probably we won't have, yes each badge would look different (colors/images/text) but that will be defined at creation time but using the same skeleton.

Reply:
we don't need them yet, but I thought we would need them in the future. But if not, I can remove them